src/uct/ib/base/ib_md.c

```diff
@@ -334,6 +340,11 @@ void *uct_ib_md_mem_handle_thread_func(void *arg)
         mr_idx++;
     }
 
+    if (ctx->access != UCT_IB_MEM_DEREG) {
+        ucs_trace("reg_mr %p..%p took %ld\n", ctx->mr[0]->addr,
```

Comment:
pls print the size as memnunits (instead of forcing user to do math..) and print the time in useconds

Reply:
User wants to know how memory got partitioned, order threads run etc...